tests/lib/test_Repo.py

```diff
@@ -24,20 +24,32 @@ def test_Repo_instantiate_types(tmp_path, variant):
     Repo(variant)
 
 
-def test_Repo_instantiate_invalid_path(tmp_path):
+def test_Repo_instantiate_invalid_path(tmp_path: str) -> None:
+    """
+    Test that the Repo class raises the correct error if instantiated with a
```

Comment:
Going along the "spec" idea: which error?

For example:

> Repo must raise `FileNotFoundError` when instantiated with a non-existent path.

Or

> Repo must be instantiated with an existing directory. Otherwise, raise `FileNotFoundError`.